### DISCUSSION TITLE
Cannot assign attribute points to Hologram character

### DISCUSSION THREAD

**Discussion by DragonEnergist** [bcholmes/StarTrek2d20#88] [Q&A] [answered]
As per the topic title, I cannot assign attribute points to Hologram character.

The plus/minus squares don't appear on the page, meaning you can't make a selection or advance to the next step.

It's not an issue if you select mixed species and choose one with fixed stats as your primary, but the error locks out the option to use Hologram as your primary/only species.

I've looked for similar issues and checked my extensions, haven't found anything.

**Comment by bcholmes** [comment, marked as answer]
Whoops. You're right. That's a bug.

**Reply by DragonEnergist** [reply]
And it looks like it's already fixed.

Awesome, thanks.
